### DISCUSSION TITLE
Personal web

### DISCUSSION THREAD

**Discussion by Temesgenchali** [community/community#50289] [Mobile]
### Select Topic Area

General

### Body

Can i create my website

**Comment by naimur-reza** [comment]
While it is possible to create a website on a mobile phone, it is generally easier to do so on a computer or laptop with a larger screen and a keyboard. However, if you are determined to create a website on your phone, here are some steps you can follow:

1. Choose a website builder: There are several website builders that offer mobile apps, such as Wix, Weebly, Squarespace, and WordPress. Download the app from the App Store or Google Play Store.

2. Sign up and choose a template: Once you have downloaded the app, sign up for an account and choose a template that you like.

3. Customize the template: Use the app's editing tools to customize the template. You can add text, images, and other elements to your website.

4. Preview and publish your website: Once you have finished customizing your website, preview it to make sure it looks the way you want it to. Then, publish it so that it is live on the internet.

5. Manage your website: You can continue to manage your website from your phone by using the app to make updates, add new content, and respond to comments.

Keep in mind that creating a website on a mobile phone may have limitations, such as limited design options and a smaller screen size, so it may be best to use a computer or laptop if possible.

**Comment by Temesgenchali** [comment]
Thank very mutch

**Reply by naimur-reza** [reply]
Make it answer brother :)

**Comment by OLLO-D-KAMBOU** [comment]
Communication -  culturel - marketing
